### PR TITLE
Add GameCollection class and tests

### PR DESCRIPTION
**Pull request by sciencefixion**
Yes!

### CODE REVIEW

**Review by gabichuelas — APPROVED** [review]
Yay GameCollection!

File: lib/game_collection.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require "csv"
+
+class GameCollection
+  def initialize(games_csv)
+    @games_csv =games_csv
```
Comment (rrabinovitch) [inline lib/game_collection.rb:5]: next person to branch should add a space before games_csv after the = sign

**Review by araspaas — APPROVED** [review]
looks good

File: test/game_collection_test.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require 'mocha/minitest'
```
Comment (rrabinovitch) [inline test/game_collection_test.rb:3]: remove mocha requirement until we need it?

**Review by rrabinovitch — APPROVED** [review]
Next person to branch should address the small details I pointed out in the code comments, but overall, looks great!

**Pull request merged**